### PR TITLE
[BM-502] Baby device - iOS doesn't disable video preview

### PR DESCRIPTION
**Pull request by anyashka**
### Ticket

https://netguru.atlassian.net/browse/BM-502
 
### Task Description

 Baby device on iOS is disabling the video preview after 60 seconds of inactivity.

### CODE REVIEW

File: Baby Monitor/Source Files/Custom Views/DisabledVideoView.swift [hunk 1]
```diff
@@ -0,0 +1,55 @@
+//
+//  DisabledVideoView.swift
+//  Baby Monitor
+//
+
+import UIKit
+
+final class DisabledVideoView: BaseView {
+
+    private(set) var tapGestureRecognizer = UITapGestureRecognizer()
```
Comment (bartosy-ymija) [inline Baby Monitor/Source Files/Custom Views/DisabledVideoView.swift]: Couldn't it be a let?

File: Baby Monitor/Resources/Constants.swift [hunk 2]
```diff
@@ -16,6 +16,9 @@ enum Constants {
 
     /// A limit per which one notification can be sent.
     static let notificationRequestTimeLimit: TimeInterval = 60
+
+    /// A time after which the video stream should be hidden.
+    static let videoStreamingHiddenAfterTime: TimeInterval = 60
```
Comment (bartosy-ymija) [inline Baby Monitor/Resources/Constants.swift]: You could consider something like `videoStreamVisibilityTimeLimit` to keep it in similar pattern as the variable above - but it's entirely optional

File: Baby Monitor/Source Files/Modules/Server/ServerViewController.swift [hunk 3]
```diff
@@ -117,4 +133,10 @@ final class ServerViewController: BaseViewController {
         localVideoTrack = stream.videoTracks[0] as? RTCVideoTrack
         localVideoTrack?.add(localView)
     }
+
+    private func fireVideoTimer() {
+        videoTimer = Observable<Int>.interval(Constants.videoStreamingHiddenAfterTime,
+                                              scheduler: MainScheduler.instance)
+            .startWith(0)
```
Comment (bartosy-ymija) [inline Baby Monitor/Source Files/Modules/Server/ServerViewController.swift]: Is `.startWith(0)` necessary? It seems that you're skipping it anyway on subscription

File: Baby Monitor/Resources/Constants.swift (hunk 2 above)
Comment (anyashka) [reply Baby Monitor/Resources/Constants.swift]: Very nice note :)

File: Baby Monitor/Source Files/Custom Views/DisabledVideoView.swift (hunk 1 above)
Comment (anyashka) [reply Baby Monitor/Source Files/Custom Views/DisabledVideoView.swift]: Of course! Thx :)

File: Baby Monitor/Source Files/Modules/Server/ServerViewController.swift (hunk 3 above)
Comment (anyashka) [reply Baby Monitor/Source Files/Modules/Server/ServerViewController.swift]: Nice catch!

**Pull request merged**